recipes/jenkins.rb

```diff
@@ -455,13 +455,138 @@ def jenkins = Jenkins.getInstance()
     GROOVY
 end
 
-file '/var/lib/jenkins/init.groovy.d/credentials_config.groovy' do
+file '/var/lib/jenkins/init.groovy.d/02-credentials_config.groovy' do
   content credentials_scripts.join("\n")
   mode '0500'
   owner 'jenkins'
   group 'jenkins'
 end
 
+file '/var/lib/jenkins/init.groovy.d/99-approved_signatures.groovy' do
+  content <<~GROOVY
```

Comment:
This is a long enough set of contents that I'd consider making it a cookbook file (or even a template) rather than embedding the file contents here.